packages/java/parser-jvm-plugin-model/src/test/resources/dev/hilla/parser/plugins/model/validation/openapi.json

```diff
@@ -138,8 +147,10 @@
             "nullable": true,
             "items": {
               "type": "string",
-              "nullable": true
+              "nullable": true,
+              "x-java-type": "java.lang.String"
             },
+            "x-java-type": "java.util.List",
```

Comment:
Should we leave generics out until we need them?

Reply:
The item type information is sort of there, in the `items` object:
```
"aStringList" : {
  "type" : "array",
  "nullable" : true,
  "items" : {
    "type" : "string",
    "nullable" : true,
    "x-java-type" : "java.lang.String"
  },
  "x-java-type" : "java.util.List"
},
```